packages/compiler/src/transforms/add-imports.ts

```diff
@@ -107,6 +107,10 @@ export type IdentifierList = Record<string, true>;
 export function findAllDanglingIdentifiers(ast: ASTNode) {
   const result: IdentifierList = {};
   visit(ast, {
+    visitObjectExpression: () => {
```

Comment:
Do the same thing for lazy state. Probably a good idea to share the transformer function?